File: 05 (Box 1)/p05.java
import java.util.Scanner;

public class Main {
    static void solve() {
        // Your solution code goes here
    }

    public static void main(String[] args) {
        // #ifndef ONLINE_JUDGE
        //     // freopen("input.txt", "r", System.in);
        //     // freopen("output.txt", "w", System.out);
        // #endif

        Scanner scanner = new Scanner(System.in);

        int tc = scanner.nextInt();

        for (int i = 0; i < tc; i++) {
            int a = scanner.nextInt();

            for (int j = 0; j < a; j++) {
                for (int k = 0; k < a; k++) {
                    System.out.print("*");
                }
                System.out.println();
            }

            System.out.println();
        }
    }
}

File: 05 (Box 1)/p05.c
#include <stdio.h>

void solve() {
    // Your solution code goes here
}

int main() {
    #ifndef ONLINE_JUDGE
        // freopen("input.txt", "r", stdin);
        // freopen("output.txt", "w", stdout);
    #endif

    int tc;
    scanf("%d", &tc);

    for (int i = 0; i < tc; i++) {
        int a;
        scanf("%d", &a);

        for (int j = 0; j < a; j++) {
            for (int k = 0; k < a; k++) {
                printf("*");
            }
            printf("\n");
        }

        printf("\n");
    }
    return 0;
}
